github/examples_test.go

```diff
@@ -74,3 +74,24 @@ func ExampleUsersService_ListAll() {
 		// Process users...
 	}
 }
+
+func ExamplePullRequestsService_Create() {
+	// In this example we're creating a PR and displaying the HTML url at the end.
+	client := github.NewClient(nil)
+
+	newPR := &github.NewPullRequest{
+		Title:               github.String("My awesome pull request"),
+		Head:                github.String("branch_to_merge"),
+		Base:                github.String("master"),
+		Body:                github.String("This is the description of the PR created with the package `github.com/google/go-github/github`"),
+		MaintainerCanModify: github.Bool(true),
+	}
+
+	pr, _, err := client.PullRequests.Create(context.Background(), "myOrganization", "myRepository", newPR)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	fmt.Printf("PR created: %s\n", *pr.HTMLURL)
```

Comment:
Sorry I didn't see this before... I meant to use the accessor here:
`pr.GetHTMLURL()`
instead of `*pr.HTMLURL`